### ISSUE TITLE
Support for jack emitter wanted

### ISSUE THREAD

**Issue by akamaus** [quiniouben/vban#52]
Currently running `vban_emitter -b jack` results in following error message:
```Fatal: Sorry jack backend is not ready for emitter yet```

Having support for it would allow live instrumental sessions over the network.

**Comment by quiniouben** [comment]
Hi @akamaus , yes this support would be really nice. In fact I dislike the way I did it for the receptor, and therefore don't want to repeat the same mistakes for an emitter implementation. Unfortunately, in general and especially in this period of confinement, I completely lack spare time to work on this. But this is in my todo list :).

**Comment by giulianobernardi** [comment]
I second the OP request. It would be great to have support for the jack backend for the emitter!
Regardless, thanks for the nice work done with this repo.

**Comment by quiniouben** [comment]
Yep, well, as you observed, spare time has not improved since last year. But let's see how things evolve

**Comment by chaseology** [comment]
Yes, Jack Support, especially with the new DEV running the latest JACK AUDIO CONNECTION KIT upgrades/updates, falkTX from KXstudio,  higher performance, optimized, full Jack support for VBAN emitter/receptor would be fantastic
